domains/chandrasekhar

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "Chandra-Sekhar-Bala",
+    "email": "sekhar.chandra.5832@gmail.com"
+  },
+  "record": {
+    "URL": "https://chandra-sekhar-bala.github.io"
```

Comment:
Site doesn't seem to be accessible, is the website hosted?

Reply:
Yup! You can check it's working.